### PR TITLE
feat: refactor input and results page components and added styling

### PR DESCRIPTION
**Pull request by pratikmdhr**
## WHAT IT DOES:

_Description of the pull request, what changed_

- Refactored components for Input page and results page and the header component
- Added styling based on new design specs
- Added [svgr](https://www.npmjs.com/package/@svgr/webpack) to import SVGs

## HOW TO TEST:

_Description of how to test if applicable_

### PREVIEW:

_Attach any relevant videos or images:_

---

### DESIGN:

_Attach link to figma file or design files here:_
[New design specs](https://www.figma.com/file/7s0eu29dLZMuYpREjAjt2k/Sum-It-Up?node-id=234%3A1290&t=K3Li2gxZknpzbGjh-4)

---

### WHAT'S LEFT:

_Delete if not necessary:_

- [ ] - Style SummaryLengthSlider component
- [ ] - Refine styling for input fields (for focus states, animate placeholders?)
- [ ] - Footer component styling and make it sticky
- [ ] - Add About page
- [ ] - Styling for the Original Content in the Results page

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -6,16 +6,26 @@ module.exports = {
...
+      "3xl": "5.4rem",
+    },
     extend: {
       fontSize: {
-        heading1: `2.75rem`,
-        heading2: `2rem`,
-        heading3: `1.5rem`,
-        heading4: `1rem`,
-        paragraph: `0.875rem`,
+        heading1: `4.4rem`,
```
Comment (pratikmdhr) [inline tailwind.config.js:20]: these could be deleted i guess, but it was on the Figma sheet

File: src/app/page.tsx [hunk 2]
```diff
@@ -1,17 +1,20 @@
 "use client";
 import { useState } from "react";
 import InputComponent from "~/components/Input/Input";
-import SongResult from "~/components/Result/SongResult";
-import TextResult from "~/components/Result/TextResult";
+import SongResult from "~/components/Result/ResultContent/SongResult";
+import TextResult from "~/components/Result/ResultContent/TextResult";
```
Comment (MaiaS) [inline src/app/page.tsx]: unused imports

File: src/app/layout.tsx [hunk 3]
```diff
@@ -11,9 +10,8 @@ export default function RootLayout({ children }: { children: React.ReactNode })
         head.tsx. Find out more at https://beta.nextjs.org/docs/api-reference/file-conventions/head
       */}
       <head />
-      <body className="flex flex-col justify-between">
+      <body className="flex flex-col justify-between text-base font-splineSans" >
         <ReactQueryWrapper>
-          <Header />
           <main className="flex-1">{children}</main>
```
Comment (MaiaS) [inline src/app/layout.tsx:15]: Noticed that the page doesn't fill the entire available height, makes the footer look a little jumpy

**Review by MaiaS — APPROVED** [review]
Thank you for this! Only thing I noticed was we should make sure the page fills the entire height.

File: src/app/layout.tsx (hunk 3 above)
Comment (pratikmdhr) [reply src/app/layout.tsx:15]: @MaiaS could you check it out once again, it should be fixed

Comment (MaiaS) [reply src/app/layout.tsx:15]: all good! Thank you for fixing it!

**Pull request merged**